server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java

```diff
@@ -192,6 +192,33 @@ public CompletableFuture<Token> updateToken(ServiceRequestContext ctx,
         );
     }
 
+    /**
+     * PATCH /tokens/{appId}/permission
+     *
+     * <p>Updates a permission of a token of the specified ID.
+     */
+    @Patch("/tokens/{appId}/level")
+    @RequiresAdministrator
+    public CompletableFuture<Token> updateTokenLevel(ServiceRequestContext ctx,
+                                                     @Param String appId,
+                                                     TokenLevelRequest tokenLevelRequest,
+                                                     Author author, User loginUser) {
+
+        return getTokenOrRespondForbidden(ctx, appId, loginUser).thenCompose(
+                token -> {
```

Comment:
Should we return `304 Not Modified` if the token is at the desired level already?

Reply:
Okay, I'll add 
```
 throw HttpStatusException.of(HttpStatus.NOT_MODIFIED);
```
when the current status and desired one is same.